data/presets/natural/tree.json

```diff
@@ -1,14 +1,20 @@
 {
     "icon": "maki-park",
     "fields": [
+        "genus",
+        "species",
+        "taxon",
         "leaf_type_singular",
         "leaf_cycle_singular",
-        "denotation",
-        "height",
-        "diameter"
+        "start_date"
     ],
     "moreFields": [
+        "source",
+        "height",
         "circumference",
+        "diameter_crown",
```

Comment:
This field is currently not yet defined, as far as I can see. You would need to create a new file in https://github.com/openstreetmap/id-tagging-schema/tree/main/data/fields for it (you can take https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/diameter.json as a template).

Reply:
Thank @tyrasd 👍  I created the file. Please review it 😃